man/Dockerfile.aarch64

```diff
@@ -1,6 +1,25 @@
-FROM    aarch64/ubuntu:xenial
+FROM     aarch64/ubuntu:wily
```

Comment:
nit: can you remove the extra space at the beginning so it's in-line with everything else?